Assets/Vimeo/Scripts/Services/VimeoApi.cs

```diff
@@ -301,36 +297,56 @@ IEnumerator Patch(string url)
                 if (request.responseCode != 200) {
                     Debug.LogError(request.downloadHandler.text);
                     if (OnError != null) OnError(request.downloadHandler.text);
-                }
-                else if (OnPatchComplete != null) {
+                } else if (OnPatchComplete != null) {
                     OnPatchComplete(request.downloadHandler.text);
                 }
             }
         }
+        public IEnumerator RequestTusResource(string api_path, long fileByteCount)
+        {
+            string tusResourceRequestBody = "{ \"upload\": { \"approach\": \"tus\", \"size\": \"" + fileByteCount.ToString() + "\" } }";
+
+            if (token != null) {
+                using (UnityWebRequest request = UnityWebRequest.Put("https://api.vimeo.com/me/videos", tusResourceRequestBody)) {
+                    PrepareTusHeaders(request);
+                    yield return VimeoApi.SendRequest(request);
 
+                    if (request.error != null) {
+                        Debug.Log(request.error);
```

Comment:
Remove the debugs

Reply:
🙌 https://github.com/vimeo/vimeo-unity-sdk/pull/46/commits/2427d90deb58cbaf846728cff0a46aec75b7e04b